FetchMigration/README.md

```diff
@@ -1,74 +1,76 @@
-# "Fetch" Data Migration / Backfill
+# Fetch  Migration and Backfill
 
-Fetch Migration provides an easy-to-use tool that simplifies the process of moving indices and their data from a 
-"source" cluster (either Elasticsearch or OpenSearch) to a "target" OpenSearch cluster. It automates the process of 
-comparing indices between the two clusters and only creates index metadata (settings and mappings) that do not already 
-exist on the target cluster. Internally, the tool uses [Data Prepper](https://github.com/opensearch-project/data-prepper) 
-to migrate data for these created indices.
+The **Fetch Migration** tool simplifies the process of moving indexes and the data contained within them from a
+source cluster, such as Elasticsearch or OpenSearch, to a target OpenSearch cluster. It automates the process of 
+comparing indexes between the two clusters by creating index metadata, like settings and mappings that does not already exist on the target cluster using [Data Prepper](https://github.com/opensearch-project/data-prepper).
```

Comment:
There are some technical inaccuracies here:
1. Data Prepper is only used for data migration, and not for index comparison and creation as this line implies
2. Indexes are compared _before_ they are created, not _by_ creating index metadata.